### ISSUE TITLE
Deployment instructions out of date?

### ISSUE THREAD

**Issue by therrealdude** [OfficeDev/microsoft-teams-apps-bookaroom#36]
I am following the deployment instructions and an issue when getting to the end of step 2 to step 3. I am not receiving a botid or appdomain as described in the instructions. I am also not seeing any option to add a setting as you describe in step 3. Has the procedure for deploying and running teams bots changed?

**Comment by therrealdude** [comment]
I was able to find the botid and domain, but I am still unable to figure out how to add a setting to enable the authentication for step 3.

**Comment by georgepetkovMT** [comment]
Yeah, i struggled with that too. You need to go to the bot page and under "Configuration" you have the option to "Add OAuth Connection Settings", from there on you follow the instructions as they are.
